qiskit_ibm_transpiler/wrappers/ai_api_synthesis.py

```diff
@@ -44,6 +44,7 @@ def transpile(
         backend: Union[Backend, None] = None,
     ):
         if coupling_map is not None:
+            logger.info("Running synthesis against the Qiskit Transpiler Service")
```

Comment:
I think we can add here the type of synthesis we are running: Clifford, Permutation, etc.

Reply:
Yes, I was thinking on doing that in other PR, I was trying to reduce this one only for tests. Do you agree? (I needed to add that line in order to pass the tests on this PR)

Reply:
Per offline conversations we agree on leave this for a following PR